Source/WebInspectorUI/UserInterface/Models/Canvas.js

```diff
@@ -147,9 +162,9 @@ WI.Canvas = class Canvas extends WI.Object
     {
         switch(colorSpace) {
         case WI.Canvas.ColorSpace.SRGB:
-            return WI.unlocalizedString("sRGB");
+            return WI.UIString("sRGB", "sRGB @ Color Space", "Label for a canvas that uses the sRGB color space.");
```

Comment:
Not sure it's correct that we begin translating these color space names. To my knowledge sRGB and Display P3 aren't typically localized. For example, the product page for Apple Studio Display in France says `Internet et Web (sRGB)` and in the US says `Internet and Web (sRGB)`.

Reply:
id rather leave that up to the localizers, as that may not always be true in every language (e.g. Japanese might prefer to use katakana instead of english)

my approach is usually "unless what we're showing is a code snippet or something from the developer, everything should have the opportunity to be localized"